docker-compose.yml

```diff
@@ -113,12 +113,15 @@ services:
       - "3000:3000"
       - "2112:2112"
     depends_on:
-      - mysql
-      - redis-server
-      - trillian-log-server
+      mysql:
+        condition: service_healthy
+      redis-server:
+        condition: service_healthy
+      trillian-log-server:
+        condition: service_started
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:3000/ping"]
-      interval: 10s
+      interval: 30s
```

Comment:
To minimize startup time, can we keep the interval low but increase retries?

Reply:
yes, changed it 10s, and 15 retries.